.gitignore

```diff
@@ -1,8 +1,7 @@
 *.iml
 .gradle
-/local.properties
-/.idea/workspace.xml
```

Comment:
I saw that the whole `.idea` folder is now git-ignored, which is the right thing.